app/src/main/assets/html/error/this-connection-is-untrusted.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>This Connection is Untrusted</title>
+    <body data-type="danger">
+        <script src="js/error.js"></script>
+        <div class="container-wide">
+            <h1>This Connection is Untrusted</h1>
+
+            <p>You have asked Firefox to connect securely to <b id="errorURL">www.abc.com</b> but we can’t confirm that your connection is secure.</p>
```

Comment:
should this be `Firefox` or [`Firefox Reality` like used elsewhere](https://github.com/MozillaReality/FirefoxReality/pull/170/files#diff-73de361bd7338b90c2f969642b098e42R13)?